commands/__init__.py

```diff
@@ -8,5 +8,6 @@
 
 See LICENSE.md
 """
-from . import *
 __all__ = ["debug"]
+
+from commands import *
```

Comment:
`import .debug` might make the point of this clearer. Also, by going with that pattern, we can in the future choose which commands to import with ifs.

Reply:
I think the cleanest approach is to do a wildcard import of all modules in commands, we can disable the loading of specific commands via logic. eg i can check for the presence of the debug mode flag before loading the actual debug commands. This would only slightly reduce startup times, but not affect runtime speeds.

As for Clarity, i think this is clear enough. it imports every module from commands into the current namespace, as defined by the `__all__` entry